src/map/clif.cpp

```diff
@@ -18591,11 +18591,10 @@ void clif_parse_reqworldinfo(int fd,struct map_session_data *sd) {
 		clif_ackworldinfo(sd);
 }
 
-/// unknown usage (CZ_BLOCKING_PLAY_CANCEL)
-/// 0447
+/// Enable playing abilities of a player if map is done loading.
+/// 0447 (CZ_BLOCKING_PLAY_CANCEL)
 void clif_parse_blocking_playcancel(int fd,struct map_session_data *sd) {
-	//if(sd)
-	;
+	status_change_end(&sd->bl, SC_BLOCKING_PLAY, INVALID_TIMER);
```

Comment:
Should we check sd warping state here?
So that we ignore faked packets from bots?